### ISSUE TITLE
Argument 1 passed to "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" must be an instance of Symfony\Component\EventDispatcher\Event, Lexik\Bundle\JWTAuthenticationBundle\Event\JWTAuthenticatedEvent given.

### ISSUE THREAD

**Issue by niepi** [lexik/LexikJWTAuthenticationBundle#657]
Hi, I'm getting the following error with Symfony 4.3

Argument 1 passed to "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" must be an instance of Symfony\Component\EventDispatcher\Event, Lexik\Bundle\JWTAuthenticationBundle\Event\JWTAuthenticatedEvent given.

**Comment by chalasr** [comment]
`EventDispatcherInterface::dispatch()` does not have the `Event` typehint in 4.3. Are you sure that you correctly installed `symfony/event-dispatcher` to `4.3`?

**Comment by niepi** [comment]
thx for the quick reply

just checked and rerun the upgrade

`  - Updating symfony/event-dispatcher (v4.2.6 => v4.3.0): Loading from cache`

**Comment by chalasr** [comment]
Can you please show the full exception trace ? Is the cache properly cleared for the environment on which you're getting the error? A `rm -rf var/cache/*` could help eliminating the doubt :)

**Comment by niepi** [comment]
i did `rm -rf var/cache/*`

stack trace


```
TypeError:
Argument 1 passed to "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" must be an instance of Symfony\Component\EventDispatcher\Event, Lexik\Bundle\JWTAuthenticationBundle\Event\JWTNotFoundEvent given.

  at vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php:154
  at Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(object(JWTNotFoundEvent), object(JWTNotFoundEvent))
     (vendor/lexik/jwt-authentication-bundle/Security/Firewall/JWTListener.php:83)
  at Lexik\Bundle\JWTAuthenticationBundle\Security\Firewall\JWTListener->handle(object(RequestEvent))
     (vendor/symfony/security-bundle/Debug/WrappedListener.php:58)
  at Symfony\Bundle\SecurityBundle\Debug\WrappedListener->__invoke(object(RequestEvent))
     (vendor/symfony/security-bundle/Debug/TraceableFirewallListener.php:35)
  at Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener->callListeners(object(RequestEvent), object(Generator))
     (vendor/symfony/security-http/Firewall.php:97)
  at Symfony\Component\Security\Http\Firewall->onKernelRequest(object(RequestEvent), 'kernel.request', object(TraceableEventDispatcher))
     (vendor/symfony/event-dispatcher/Debug/WrappedListener.php:126)
  at Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(object(RequestEvent), 'kernel.request', object(TraceableEventDispatcher))
     (vendor/symfony/event-dispatcher/EventDispatcher.php:260)
  at Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(array(object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener)), 'kernel.request', object(RequestEvent))
     (vendor/symfony/event-dispatcher/EventDispatcher.php:235)
  at Symfony\Component\EventDispatcher\EventDispatcher->callListeners(array(object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener)), 'kernel.request', object(RequestEvent))
     (vendor/symfony/event-dispatcher/EventDispatcher.php:75)
  at Symfony\Component\EventDispatcher\EventDispatcher->dispatch(object(RequestEvent), 'kernel.request')
     (vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php:168)
  at Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(object(RequestEvent), 'kernel.request')
     (vendor/symfony/http-kernel/HttpKernel.php:127)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:68)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:198)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (public/index.php:37)
```

**Comment by LemosT** [comment]
Also having the exact same problem.

**Comment by chalasr** [comment]
I will investigate this week, it might be a symfony bug.

**Comment by mickaelottmann** [comment]
same issue

**Comment by Dinath** [comment]
Same here since update from `v4.2.4 => v4.3.1`. Bonne chance.

**Comment by arifsetianto** [comment]
I had same issue, is there a solution for this?

**Comment by KDederichs** [comment]
I have the same one but for the AuthenticationSuccessEvent, I assume it's related?

`Argument 1 passed to "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" must be an instance of Symfony\Component\EventDispatcher\Event, Lexik\Bundle\JWTAuthenticationBundle\Event\AuthenticationSuccessEvent given.`

**Comment by lucascardoso-egsys** [comment]
Same issue here. Any idea how can i fix it?

**Comment by lucascardoso-egsys** [comment]
I fixed this error. It occurred for me because i was overriding the AuthenticationSuccessHandler.

So I need copy again this classe from: https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/Security/Http/Authentication/AuthenticationSuccessHandler.php

and apply my changes again.

I hope it helps someone, cheers!!

**Comment by KDederichs** [comment]
Turned out to be related to something else in my codebase sorry

**Comment by mickaelottmann** [comment]
`CRITICAL20:25:56 | php | Argument 1 passed to "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" must be an instance of Symfony\Component\EventDispatcher\Event, Lexik\Bundle\JWTAuthenticationBundle\Event\JWTAuthenticatedEvent given.Show context    Hide trace{▼   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php:154 {▶}   /var/www/html/backend/vendor/lexik/jwt-authentication-bundle/Security/Authentication/Provider/JWTProvider.php:93 {▶}   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/Security/Core/Authentication/AuthenticationProviderManager.php:80 {▶}   /var/www/html/backend/vendor/lexik/jwt-authentication-bundle/Security/Firewall/JWTListener.php:96 {▶}   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Bundle/SecurityBundle/Debug/WrappedListener.php:58 {▶}   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Bundle/SecurityBundle/Debug/TraceableFirewallListener.php:35 {▶}   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/Security/Http/Firewall.php:97 {▶}   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/WrappedListener.php:126 {▶}   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:260 {▶}   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:235 {▶}   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:75 {▶}   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php:168 {▶}   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:127 {▶}   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:68 {▶}   /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:198 {▶}   /var/www/html/backend/web/app_dev.php:14 {▼     › $request = Request::createFromGlobals();     › $response = $kernel->handle($request);     › $response->send();     arguments: {▼       $request: Request {#53 …}     }   } }
-- | -- | --`

`Uncaught PHP Exception TypeError: "Argument 1 passed to "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" must be an instance of Symfony\Component\EventDispatcher\Event, Lexik\Bundle\JWTAuthenticationBundle\Event\JWTAuthenticatedEvent given." at /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php line 154
Hide context    Hide trace
[▼
  "exception" => FatalThrowableError^ {#331 ▼
    -originalClassName: "TypeError"
    #message: "Argument 1 passed to "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" must be an instance of Symfony\Component\EventDispatcher\Event, Lexik\Bundle\JWTAuthenticationBundle\Event\JWTAuthenticatedEvent given."
    #code: 0
    #file: "/var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php"
    #line: 154
    #severity: E_RECOVERABLE_ERROR
    trace: {▼
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php:154 {▶}
      /var/www/html/backend/vendor/lexik/jwt-authentication-bundle/Security/Authentication/Provider/JWTProvider.php:93 {▶}
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/Security/Core/Authentication/AuthenticationProviderManager.php:80 {▶}
      /var/www/html/backend/vendor/lexik/jwt-authentication-bundle/Security/Firewall/JWTListener.php:96 {▶}
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Bundle/SecurityBundle/Debug/WrappedListener.php:58 {▶}
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Bundle/SecurityBundle/Debug/TraceableFirewallListener.php:35 {▶}
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/Security/Http/Firewall.php:97 {▶}
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/WrappedListener.php:126 {▶}
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:260 {▶}
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:235 {▶}
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:75 {▶}
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php:168 {▶}
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:127 {▶}
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:68 {▶}
      /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:198 {▶}
      /var/www/html/backend/web/app_dev.php:14 {▼
        › $request = Request::createFromGlobals();
        › $response = $kernel->handle($request);
        › $response->send();
        arguments: {▶}
      }
    }
  }
]
{▼
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php:154 {▶}
  /var/www/html/backend/vendor/lexik/jwt-authentication-bundle/Security/Authentication/Provider/JWTProvider.php:93 {▶}
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/Security/Core/Authentication/AuthenticationProviderManager.php:80 {▶}
  /var/www/html/backend/vendor/lexik/jwt-authentication-bundle/Security/Firewall/JWTListener.php:96 {▶}
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Bundle/SecurityBundle/Debug/WrappedListener.php:58 {▶}
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Bundle/SecurityBundle/Debug/TraceableFirewallListener.php:35 {▶}
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/Security/Http/Firewall.php:97 {▶}
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/WrappedListener.php:126 {▶}
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:260 {▶}
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:235 {▶}
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:75 {▶}
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php:168 {▶}
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:127 {▶}
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:68 {▶}
  /var/www/html/backend/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:198 {▶}
  /var/www/html/backend/web/app_dev.php:14 {▼
    › $request = Request::createFromGlobals();
    › $response = $kernel->handle($request);
    › $response->send();
    arguments: {▶}
  }
}`

**Comment by mickaelottmann** [comment]
I solved this issue by removing the first argument in 
vendor/lexik/jwt-authentication-bundle/Security/Authentication/Provider/JWTProvider.php
line 93
but not sure it's the best solution, hope I helped you

**Comment by Evgeny1973** [comment]
Try  `$dispatcher->dispatch($event, OrderPlacedEvent::NAME);`

**Comment by remoteclient** [comment]
I also get this error when I try to login with bad credentials.

**Comment by MLDMoritz** [comment]
Same here, running Sf 4.3.1

I append my stacktrace if it helps:

```json
{
  "@context": "/api/contexts/Error",
  "@type": "hydra:Error",
  "hydra:title": "An error occurred",
  "hydra:description": "Argument 1 passed to \"Symfony\\Component\\EventDispatcher\\EventDispatcherInterface::dispatch()\" must be an instance of Symfony\\Component\\EventDispatcher\\Event, Lexik\\Bundle\\JWTAuthenticationBundle\\Event\\AuthenticationSuccessEvent given.",
  "trace": [
    {
      "namespace": "",
      "short_class": "",
      "class": "",
      "type": "",
      "function": "",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/messenger/Middleware/HandleMessageMiddleware.php",
      "line": 82,
      "args": []
    },
    {
      "namespace": "Symfony\\Component\\Messenger\\Middleware",
      "short_class": "HandleMessageMiddleware",
      "class": "Symfony\\Component\\Messenger\\Middleware\\HandleMessageMiddleware",
      "type": "->",
      "function": "handle",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/messenger/Middleware/SendMessageMiddleware.php",
      "line": 81,
      "args": [
        [
          "object",
          "Symfony\\Component\\Messenger\\Envelope"
        ],
        [
          "object",
          "Symfony\\Component\\Messenger\\Middleware\\TraceableStack"
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\Messenger\\Middleware",
      "short_class": "SendMessageMiddleware",
      "class": "Symfony\\Component\\Messenger\\Middleware\\SendMessageMiddleware",
      "type": "->",
      "function": "handle",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/messenger/Middleware/FailedMessageProcessingMiddleware.php",
      "line": 36,
      "args": [
        [
          "object",
          "Symfony\\Component\\Messenger\\Envelope"
        ],
        [
          "object",
          "Symfony\\Component\\Messenger\\Middleware\\TraceableStack"
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\Messenger\\Middleware",
      "short_class": "FailedMessageProcessingMiddleware",
      "class": "Symfony\\Component\\Messenger\\Middleware\\FailedMessageProcessingMiddleware",
      "type": "->",
      "function": "handle",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/messenger/Middleware/DispatchAfterCurrentBusMiddleware.php",
      "line": 67,
      "args": [
        [
          "object",
          "Symfony\\Component\\Messenger\\Envelope"
        ],
        [
          "object",
          "Symfony\\Component\\Messenger\\Middleware\\TraceableStack"
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\Messenger\\Middleware",
      "short_class": "DispatchAfterCurrentBusMiddleware",
      "class": "Symfony\\Component\\Messenger\\Middleware\\DispatchAfterCurrentBusMiddleware",
      "type": "->",
      "function": "handle",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/messenger/Middleware/AddBusNameStampMiddleware.php",
      "line": 39,
      "args": [
        [
          "object",
          "Symfony\\Component\\Messenger\\Envelope"
        ],
        [
          "object",
          "Symfony\\Component\\Messenger\\Middleware\\TraceableStack"
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\Messenger\\Middleware",
      "short_class": "AddBusNameStampMiddleware",
      "class": "Symfony\\Component\\Messenger\\Middleware\\AddBusNameStampMiddleware",
      "type": "->",
      "function": "handle",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/messenger/Middleware/TraceableMiddleware.php",
      "line": 45,
      "args": [
        [
          "object",
          "Symfony\\Component\\Messenger\\Envelope"
        ],
        [
          "object",
          "Symfony\\Component\\Messenger\\Middleware\\TraceableStack"
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\Messenger\\Middleware",
      "short_class": "TraceableMiddleware",
      "class": "Symfony\\Component\\Messenger\\Middleware\\TraceableMiddleware",
      "type": "->",
      "function": "handle",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/messenger/MessageBus.php",
      "line": 73,
      "args": [
        [
          "object",
          "Symfony\\Component\\Messenger\\Envelope"
        ],
        [
          "object",
          "Symfony\\Component\\Messenger\\Middleware\\TraceableStack"
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\Messenger",
      "short_class": "MessageBus",
      "class": "Symfony\\Component\\Messenger\\MessageBus",
      "type": "->",
      "function": "dispatch",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/messenger/TraceableMessageBus.php",
      "line": 43,
      "args": [
        [
          "object",
          "App\\Dto\\User\\Register"
        ],
        [
          "array",
          []
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\Messenger",
      "short_class": "TraceableMessageBus",
      "class": "Symfony\\Component\\Messenger\\TraceableMessageBus",
      "type": "->",
      "function": "dispatch",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/api-platform/core/src/Bridge/Symfony/Messenger/DataPersister.php",
      "line": 78,
      "args": [
        [
          "object",
          "App\\Dto\\User\\Register"
        ]
      ]
    },
    {
      "namespace": "ApiPlatform\\Core\\Bridge\\Symfony\\Messenger",
      "short_class": "DataPersister",
      "class": "ApiPlatform\\Core\\Bridge\\Symfony\\Messenger\\DataPersister",
      "type": "->",
      "function": "persist",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/api-platform/core/src/Bridge/Symfony/Bundle/DataPersister/TraceableChainDataPersister.php",
      "line": 56,
      "args": [
        [
          "object",
          "App\\Dto\\User\\Register"
        ],
        [
          "array",
          {
            "resource_class": [
              "string",
              "App\\Dto\\User\\Register"
            ],
            "collection_operation_name": [
              "string",
              "post"
            ],
            "receive": [
              "boolean",
              true
            ],
            "respond": [
              "boolean",
              true
            ],
            "persist": [
              "boolean",
              true
            ]
          }
        ]
      ]
    },
    {
      "namespace": "ApiPlatform\\Core\\Bridge\\Symfony\\Bundle\\DataPersister",
      "short_class": "TraceableChainDataPersister",
      "class": "ApiPlatform\\Core\\Bridge\\Symfony\\Bundle\\DataPersister\\TraceableChainDataPersister",
      "type": "->",
      "function": "persist",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/api-platform/core/src/EventListener/WriteListener.php",
      "line": 73,
      "args": [
        [
          "object",
          "App\\Dto\\User\\Register"
        ],
        [
          "array",
          {
            "resource_class": [
              "string",
              "App\\Dto\\User\\Register"
            ],
            "collection_operation_name": [
              "string",
              "post"
            ],
            "receive": [
              "boolean",
              true
            ],
            "respond": [
              "boolean",
              true
            ],
            "persist": [
              "boolean",
              true
            ]
          }
        ]
      ]
    },
    {
      "namespace": "ApiPlatform\\Core\\EventListener",
      "short_class": "WriteListener",
      "class": "ApiPlatform\\Core\\EventListener\\WriteListener",
      "type": "->",
      "function": "onKernelView",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/event-dispatcher/Debug/WrappedListener.php",
      "line": 126,
      "args": [
        [
          "object",
          "Symfony\\Component\\HttpKernel\\Event\\ViewEvent"
        ],
        [
          "string",
          "kernel.view"
        ],
        [
          "object",
          "Symfony\\Component\\HttpKernel\\Debug\\TraceableEventDispatcher"
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\EventDispatcher\\Debug",
      "short_class": "WrappedListener",
      "class": "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener",
      "type": "->",
      "function": "__invoke",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/event-dispatcher/EventDispatcher.php",
      "line": 260,
      "args": [
        [
          "object",
          "Symfony\\Component\\HttpKernel\\Event\\ViewEvent"
        ],
        [
          "string",
          "kernel.view"
        ],
        [
          "object",
          "Symfony\\Component\\HttpKernel\\Debug\\TraceableEventDispatcher"
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\EventDispatcher",
      "short_class": "EventDispatcher",
      "class": "Symfony\\Component\\EventDispatcher\\EventDispatcher",
      "type": "->",
      "function": "doDispatch",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/event-dispatcher/EventDispatcher.php",
      "line": 235,
      "args": [
        [
          "array",
          [
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ]
          ]
        ],
        [
          "string",
          "kernel.view"
        ],
        [
          "object",
          "Symfony\\Component\\HttpKernel\\Event\\ViewEvent"
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\EventDispatcher",
      "short_class": "EventDispatcher",
      "class": "Symfony\\Component\\EventDispatcher\\EventDispatcher",
      "type": "->",
      "function": "callListeners",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/event-dispatcher/EventDispatcher.php",
      "line": 75,
      "args": [
        [
          "array",
          [
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ],
            [
              "object",
              "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener"
            ]
          ]
        ],
        [
          "string",
          "kernel.view"
        ],
        [
          "object",
          "Symfony\\Component\\HttpKernel\\Event\\ViewEvent"
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\EventDispatcher",
      "short_class": "EventDispatcher",
      "class": "Symfony\\Component\\EventDispatcher\\EventDispatcher",
      "type": "->",
      "function": "dispatch",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php",
      "line": 168,
      "args": [
        [
          "object",
          "Symfony\\Component\\HttpKernel\\Event\\ViewEvent"
        ],
        [
          "string",
          "kernel.view"
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\EventDispatcher\\Debug",
      "short_class": "TraceableEventDispatcher",
      "class": "Symfony\\Component\\EventDispatcher\\Debug\\TraceableEventDispatcher",
      "type": "->",
      "function": "dispatch",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/http-kernel/HttpKernel.php",
      "line": 156,
      "args": [
        [
          "object",
          "Symfony\\Component\\HttpKernel\\Event\\ViewEvent"
        ],
        [
          "string",
          "kernel.view"
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\HttpKernel",
      "short_class": "HttpKernel",
      "class": "Symfony\\Component\\HttpKernel\\HttpKernel",
      "type": "->",
      "function": "handleRaw",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/http-kernel/HttpKernel.php",
      "line": 68,
      "args": [
        [
          "object",
          "Symfony\\Component\\HttpFoundation\\Request"
        ],
        [
          "integer",
          1
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\HttpKernel",
      "short_class": "HttpKernel",
      "class": "Symfony\\Component\\HttpKernel\\HttpKernel",
      "type": "->",
      "function": "handle",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/vendor/symfony/http-kernel/Kernel.php",
      "line": 198,
      "args": [
        [
          "object",
          "Symfony\\Component\\HttpFoundation\\Request"
        ],
        [
          "integer",
          1
        ],
        [
          "boolean",
          true
        ]
      ]
    },
    {
      "namespace": "Symfony\\Component\\HttpKernel",
      "short_class": "Kernel",
      "class": "Symfony\\Component\\HttpKernel\\Kernel",
      "type": "->",
      "function": "handle",
      "file": "/home/console/PhpstormProjects/ApiPlatform/api/public/index.php",
      "line": 39,
      "args": [
        [
          "object",
          "Symfony\\Component\\HttpFoundation\\Request"
        ]
      ]
    }
  ]
}
```

**Comment by acividiniwellnet** [comment]
I solved patching this way, but I understand this needs further analysis to be fixed in a better way

```
<?php

namespace Lexik\Bundle\JWTAuthenticationBundle\Event;

use Symfony\Component\EventDispatcher\Event as BaseEvent;
use Symfony\Component\EventDispatcher\EventDispatcher;
use Symfony\Contracts\EventDispatcher\Event as ContractsBaseEvent;
use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;

class Event extends BaseEvent
{
}

```

**Comment by Webonaute** [comment]
https://github.com/lexik/LexikJWTAuthenticationBundle/pull/669 should fix your issue

**Comment by Aylarius** [comment]
While waiting for the fix to be merged, downgrading `lexik/jwt-authentication-bundle` to `v2.6.1` worked out for me.

**Comment by sercul** [comment]
I also have this issue

**Comment by jlaittom** [comment]
I have the same problem

**Comment by anjanasilva** [comment]
> While waiting for the fix to be merged, downgrading `lexik/jwt-authentication-bundle` to `v2.6.1` worked out for me.

Yes this did the trick. Thanks for the sage answer :).

**Comment by chalasr** [comment]
Fixed by #669
